src/main/java/com/project/model/Application.java

```diff
@@ -21,53 +22,53 @@
 
     @NotBlank(message = "Vardas negali būti tuščias")
     @Size(max = 255, message = "Vardas negali būti ilgesnis nei 255 simboliai")
-    @Column(name="full_name")
+    @Column(name = "full_name")
     private String name;
```

Comment:
If this is a full name then your variable should reflect that. It should not be `name`. Should be `fullName`.

Reply:
changed several fields' names in Application class to better reflect what values are held in them